docs/develop/api-reference/index.mdx

```diff
@@ -445,3 +450,7 @@ raise response.error if response.error?
   `}
   cli={`momento cache delete --key test-key --value test-value`}
 />
+
+### Collection data types (CDTs)
```

Comment:
Probably should capitalize Data Types since it's used in an acronym

Reply:
Yes, but not stylistically correct.